### PR TITLE
DRAFT: Pilot 1828: Add file type as part of Redis key

### PR DESCRIPTION
**Pull request by QXgu**
## Summary

Pilot 1828: Add file type as part of Redis key

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1898

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [x] Yes
- [ ] No

## Test Directions

(Additional instructions for how to run tests or validate functionality if not covered by unit tests)

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Settings(BaseSettings):
 
     APP_NAME: str = 'dataops_service'
     VERSION = '2.0.0'
-    PORT: int = 5063
+    PORT: int = 8001
```
Comment (afmam) [inline config.py:54]: Is there reason why port was changed?

File: models/file_ops_models.py [hunk 2]
```diff
@@ -60,7 +61,7 @@ class FileOperationsPOSTResponse(APIResponse):
                 'status': 'PENDING',
                 'project_code': 'testproject',
                 'operator': 'admin',
-                'progress': 0,
+                'target_type': "batch",
```
Comment (afmam) [inline models/file_ops_models.py:64]: Does "batch" indicate to front-end combination of file/folders in a copy request? If so, how do they differentiate between them?

**Review by afmam — CHANGES_REQUESTED** [review]
Left 2 comments

File: config.py (hunk 1 above)
Comment (QXgu) [reply config.py:54]: For local test though, I will change it back!

File: models/file_ops_models.py (hunk 2 above)
Comment (QXgu) [reply models/file_ops_models.py:64]: The batch will be the situation that multiple files and folders are selected at once and trigger one pipeline. The frontend will have a different logo for that

**Comment by QXgu** [comment]
@afmam Changes back now. Please check

**Review by afmam — APPROVED** [review]

**Comment by erikdvlp** [comment]
Deprecated.

**Pull request closed without merging**